src/Compilers/Core/Portable/DiagnosticAnalyzer/DiagnosticQueue.cs

```diff
@@ -130,19 +134,18 @@ public override void EnqueueNonLocal(Diagnostic diagnostic, DiagnosticAnalyzer a
                 EnqueueCore(ref _lazyNonLocalDiagnostics, diagnostic, analyzer);
             }
 
-            private void EnqueueCore(ref Dictionary<DiagnosticAnalyzer, SimpleDiagnosticQueue> lazyDiagnosticsMap, Diagnostic diagnostic, DiagnosticAnalyzer analyzer)
+            private void EnqueueCore(ref Dictionary<DiagnosticAnalyzer, SimpleDiagnosticQueue>? lazyDiagnosticsMap, Diagnostic diagnostic, DiagnosticAnalyzer analyzer)
```

Comment:
Not sure if anything needs this, but you can add a post-condition here if you want.

```suggestion
            private void EnqueueCore([NotNull] ref Dictionary<DiagnosticAnalyzer, SimpleDiagnosticQueue>? lazyDiagnosticsMap, Diagnostic diagnostic, DiagnosticAnalyzer analyzer)
```